docs/concepts/services-networking/service.md

```diff
@@ -462,6 +462,8 @@ Special notes for Azure: To use user-specified public type `loadBalancerIP`, a s
 public IP address resource needs to be created first, and it should be in the same resource
 group of the cluster. Then you could specify the assigned IP address as `loadBalancerIP`.
 
+Special notes for AWS: AWS ELB only know instance, it doesn't know POD. When you use k8s service with `type: LoadBalancer`, kube-proxy will listen on a randomly port on instance in the cluster, an ELB is created and mapped to each instances ports even if a pod, which serve service, is not present. ELB uses round robin for the loabalancing mecanism. kube-proxy (on behalf of srevice) use a random connection 
```

Comment:
suggestions:

- wrap your lines at about 80 characters boundary so the PR is easy to review/comment
- check the `Allow edits from maintainers` button so some native speaker reviewers can do a quick edit on your PR
- be careful about some obvious typos before pushing your PR

Reply:
Thanks @tengqm I've take into account your remarks for future PR